<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Producers</class>
 <widget class="QWidget" name="Producers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Поставщики</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelItemName">
       <property name="text">
        <string>Наименование</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditItemName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelTin">
       <property name="text">
        <string>ИНН</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditItemTin">
       <property name="inputMask">
        <string>9999999999</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelItemAddress">
       <property name="text">
        <string>Адрес</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditItemAddress"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelItemPhone">
       <property name="text">
        <string>Телефон</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditItemPhone">
       <property name="inputMask">
        <string>8(9999)99-99-99</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonItemInsert">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutEditing">
     <item>
      <widget class="QPushButton" name="pushButtonItemUpdate">
       <property name="text">
        <string>Обновить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonItemUpdateCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
